CONTRIBUTORS.txt

```diff
@@ -88,8 +88,8 @@ Committed Code.
 Licensing Exceptions
```

Comment:
I think we can strike this entire paragraph. There is no `docs/` subdirectory in this repo.

Reply:
It is mentioned earlier on in the document. I don't want to modify the document. Since there is no docs, this section can be disregarded anyway.

Reply:
I recall that I copy-pasta'd from [Pyramid's](https://github.com/Pylons/pyramid/blob/master/CONTRIBUTORS.txt#L46). Let's remove it, because it just adds confusion. And it would still be consistent with the earlier mention.